splice/queries/adgroup.py

```diff
@@ -0,0 +1,82 @@
+from splice.models import Adgroup
+from sqlalchemy.sql import exists
+from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy.exc import InvalidRequestError
+
+from splice.queries.common import row_to_dict
+
+
+def get_adgroups_by_campaign_id(campaign_id):
+    from splice.environment import Environment
+
+    env = Environment.instance()
+
+    rows = (
+        env.db.session
+        .query(Adgroup)
+        .filter(Adgroup.campaign_id == campaign_id)
+        .order_by(Adgroup.id.asc())
```

Comment:
That's correct. How to sort the adgroups is largely up to the UI logic.
Here we just return all the adgroups in ascending order, which has the same order as let the old ones go first.